utilities/tools/Test-ModuleLocally.ps1

```diff
@@ -263,10 +263,10 @@ function Test-ModuleLocally {
                 if ($DeploymentTest) {
                     $functionInput['retryLimit'] = 1 # Overwrite default of 3
                     # Loop through test parameter files
-                    foreach ($paramFilePath in $moduleParameterFiles) {
-                        Write-Verbose ('Deploy module [{0}] with parameter file [{1}]' -f $ModuleName, (Split-Path $paramFilePath -Leaf)) -Verbose
-                        if ($PSCmdlet.ShouldProcess(('Module [{0}] with parameter file [{1}]' -f $ModuleName, (Split-Path $paramFilePath -Leaf)), 'Deploy')) {
-                            New-TemplateDeployment @functionInput -ParameterFilePath $paramFilePath
+                    foreach ($moduleTestFile in $moduleTestFiles) {
+                        Write-Verbose ('Deploy module [{0}] with parameter file [{1}]' -f $ModuleName, (Split-Path $moduleTestFile -Leaf)) -Verbose
+                        if ($PSCmdlet.ShouldProcess(('Module [{0}] with parameter file [{1}]' -f $ModuleName, (Split-Path $moduleTestFile -Leaf)), 'Deploy')) {
+                            New-TemplateDeployment @functionInput -ParameterFilePath $moduleTestFile
```

Comment:
It looks like we renamed all parameter file occurrences in test occurrences. Should we also update the parameter name for this function (`ParameterFilePath`) ?

Reply:
You mean re-name `-ParameterFilePath` to e.g. `-TestFilePath`? 

In this case I'd say now, as the function on it's own actually wants a ParameterFile (as per the New-Az*Deployment invocation). The reason we renamed it for our modules was, that there they are 'test files' with currently JSON-ARM (and soon Bicep).

On that note: That's the reason I did not rename anything about the dependency pipeline. It's still referencing parameter files - which they absolutly are in that case :)